tasks/kernel_matrix_testing/selftest.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import annotations
+
+import functools
+import re
+from pathlib import Path
+from typing import Optional, Tuple
+
+from invoke.context import Context
+
+from tasks.kernel_matrix_testing.platforms import get_platforms
+from tasks.kernel_matrix_testing.tool import error, full_arch, get_binary_target_arch, info, warn
+from tasks.kernel_matrix_testing.types import Component
+from tasks.kernel_matrix_testing.vars import arch_ls
+
+SelftestResult = Tuple[Optional[bool], str]
+
+
+def selftest_pulumi(ctx: Context, _: bool) -> SelftestResult:
```

Comment:
consider fixing the required version / versions range in the selftest (see inv [setup](https://github.com/DataDog/datadog-agent/blob/eab9ad9595ca2caccc45d544b1243b6c4f949479/tasks/setup.py#L27) for example)

Reply:
Do we have a required version range for pulumi? I am looking at the [test-infra-definitions setup](https://github.com/DataDog/test-infra-definitions/blob/main/tasks/setup.py#L72) and it seems we just download the most recent version.